src/sentry/models/grouphashmetadata.py

```diff
@@ -0,0 +1,64 @@
+from typing import Any
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, JSONField, Model, region_silo_model
+
+
+@region_silo_model
+class GroupHashMetadata(Model):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    class GroupingMethod(models.TextChoices):
+        HASH = "hash"
+        SEER = "seer"
+
+    class HashBasis(models.TextChoices):
+        MESSAGE = "msg"
+        FULL_STACKTRACE = "stack"
+        APP_STACKTRACE = "app_stack"
+        FINGERPRINT = "fingerprint"
+        # TODO: Are there others?
+
+    # GENERAL
+    grouphash = FlexibleForeignKey("sentry.GroupHash", unique=True, related_name="metadata")
+    date_added = models.DateTimeField(default=timezone.now)
+    grouping_method = models.PositiveIntegerField(choices=GroupingMethod)
+
+    # HASHING
+    # Most recent config to produce this hash
+    # TODO: do we need first, also?
+    # TODO: Is there a way to get choices for this from grouping.strategies.configurations.CONFIGURATIONS?
+    grouping_config = models.CharField()
+    # Eg. message, in-app stacktrace, full stacktrace, fingerprint, etc
+    hash_basis = models.CharField(choices=HashBasis, null=True)
+    # Eg. if message parameterized, what type of parameterization?, etc)
+    hashing_metadata: models.Field[dict[str, Any] | None, dict[str, Any] | None] = JSONField(
```

Comment:
Do you have an example to help me understand?

Reply:
There are examples of all of these in [the Notion doc](https://www.notion.so/sentry/GroupHash-metadata-a5a41b2a7e36401182a8eaa76f66c192?pvs=4), but one possibility for this field might be:

```json
{
  "message_parametrized": true,
  "parameterization_types": ["int", "date"]
}
```